### PR TITLE
Fix soft delete check, null check

### PR DESCRIPTION
**Pull request by hi019**
## Change Summary
- Fixes @manavo 's comments in https://github.com/typesense/laravel-scout-typesense-driver/pull/15#pullrequestreview-822556807

### CODE REVIEW

File: src/Engines/TypesenseEngine.php [hunk 1]
```diff
@@ -78,7 +79,7 @@ public function update($models): void
     {
         $collection = $this->typesense->getCollectionIndex($models->first());
 
-        if ($this->usesSoftDelete($models->first()) && $models->first()->softDelete) {
+        if ($this->usesSoftDelete($models->first()) && Config::get('scout.soft_delete', false)) {
```
Comment (manavo) [inline src/Engines/TypesenseEngine.php:82]: Would recommend the “config” helper here instead of the facade

**Comment by manavo** [comment]
Looks good to me

**Comment by manavo** [comment]
@hi019 can we tag a new release with these changes? Thanks!

**Comment by hi019** [comment]
Yep, will do it later today

**Comment by manavo** [comment]
Thanks!

**Comment by hi019** [comment]
Released

**Comment by manavo** [comment]
Thanks!

**Comment by hi019** [comment]
By the way, would you be interested in becoming a maintainer of this repository @manavo ?

**Comment by manavo** [comment]
Thanks for the offer @hi019, really appreciate it! Unfortunately it's not something I can commit to at the moment, but if I have more time in the future would love to discuss again!

**Comment by hi019** [comment]
Sounds good!

**Pull request merged**